models/demos/bert_tiny/tests/perf_e2e_bert_tiny.py

```diff
@@ -0,0 +1,320 @@
+    profiler.end("compile")
+    ttnn.dump_device_profiler(device)
+
+    profiler.start("cache")
+    test_infra.input_tensor = tt_inputs_host.to(device, input_mem_config)
+    _ = ttnn.from_device(test_infra.run(), blocking=True)
+    profiler.end("cache")
+    ttnn.dump_device_profiler(device)
+
+    for iter in range(0, num_warmup_iterations):
+        test_infra.input_tensor = tt_inputs_host.to(device, input_mem_config)
+        _ = ttnn.from_device(test_infra.run(), blocking=True)
+        ttnn.dump_device_profiler(device)
+
+    ttnn.synchronize_devices(device)
+    if use_signpost:
+        signpost(header="start")
+    outputs = []
+    profiler.start(f"run")
+    for iter in range(0, num_measurement_iterations):
+        test_infra.input_tensor = tt_inputs_host.to(device, input_mem_config)
+        outputs.append(ttnn.from_device(test_infra.run(), blocking=False))
+    ttnn.synchronize_devices(device)
+    profiler.end(f"run")
+    if use_signpost:
+        signpost(header="stop")
+    ttnn.dump_device_profiler(device)
+
+
+def run_trace_2cq_model(
+    device,
+    tt_inputs,
+    token_type_ids,
+    position_ids,
+    attention_mask,
+    test_infra,
+    num_warmup_iterations,
+    num_measurement_iterations,
+):
+    (
+        tt_inputs_host,
+        tt_token_type_ids_host,
+        tt_position_ids_host,
+        tt_attention_mask_host,
+    ) = test_infra.setup_dram_sharded_input(device)
+    tt_input_ids = tt_inputs_host.to(device)
+    tt_token_type_ids = tt_token_type_ids_host.to(device)
+    tt_position_ids = tt_position_ids_host.to(device)
+    tt_attention_mask = tt_attention_mask_host.to(device)
+
+    op_event = ttnn.create_event(device)
+    write_event = ttnn.create_event(device)
+    # Initialize the op event so we can write
+    ttnn.record_event(0, op_event)
+
+    profiler.start("compile")
+    ttnn.wait_for_event(1, op_event)
+    ttnn.copy_host_to_device_tensor(tt_inputs_host, tt_input_ids, 1)
+    ttnn.copy_host_to_device_tensor(tt_token_type_ids_host, tt_token_type_ids, 1)
+    ttnn.copy_host_to_device_tensor(tt_position_ids_host, tt_position_ids, 1)
+    ttnn.copy_host_to_device_tensor(tt_attention_mask_host, tt_attention_mask, 1)
+
+    ttnn.record_event(1, write_event)
+    ttnn.wait_for_event(0, write_event)
+    test_infra.input_tensor = tt_input_ids  # ttnn.to_memory_config(tt_input_ids, input_mem_config)
+    test_infra.token_type_ids = tt_token_type_ids
+    test_infra.position_ids = tt_position_ids
+    test_infra.attention_mask = tt_attention_mask
+    shape = test_infra.input_tensor.shape
+    dtype = test_infra.input_tensor.dtype
+    layout = test_infra.input_tensor.layout
+    ttnn.record_event(0, op_event)
+    _ = ttnn.from_device(test_infra.run(), blocking=True)
+    profiler.end("compile")
+    ttnn.dump_device_profiler(device)
+
+    profiler.start("cache")
+    ttnn.wait_for_event(1, op_event)
+    # ttnn.copy_host_to_device_tensor(tt_inputs_host, tt_input_ids, 1)
```

Comment:
Don't commit commented code. 

I won't bother commenting on every single line, but there are a bunch of places in this PR where you need to get rid of the commented code.

Reply:
Removed all the commented code and updated the commit.